learning-bitmark/working-with-bitmarks/creating-bitmark-account.md

```diff
@@ -0,0 +1,122 @@
+# Creating Bitmark account
+
+<br>
+<br>
+## Bitmark account number
+
+Property owners in Bitmark system are identified by their Ed25519 public keys. Those public keys are represented by the Bitmark account numbers which are in based58 endcoded format.
+
+* Account number - livenet example:
+
+    `bqSUHTVRYnrUPBEU48riv9UwDmdRnHm9Mf9LWYuYEa7JKtqgKw`
+
+* Account number - testnet example:
+
+    `fABCJxXc8aYGoj1yLLXmsGdWEo1Y5cZE9Ko5DrHhy4HvgGYMAu`
+
+<br>
+<br>
+## Creating Bitmark account using Bitmark App
+
+* Download and install the [Android](https://apps.apple.com/us/app/bitmark-property-registry/id1429427796) or [iOS](https://apps.apple.com/us/app/bitmark-property-registry/id1429427796) Bitmark App
+
+* Open the app
+
+* Follow the instructions to create a new account
+
+* Check for the account number at Account > Settings screen
+
+* Backup the recovery phrase via Account > Settings screen > Write down recovery phrase
+
+<br>
+<br>
+## Creating Bitmark account using Bitmark SDK
+
+In this section we are using the example of **Bitmark JS SDK** to create a new Bitmark account.<br>
+For further functions and other languages, please look at the [Bitmark SDK](https://github.com/bitmark-inc/docs/blob/master/bitmark-references/bitmark-sdk/bitmark-sdk-document.md) section.
+
+* Install Bitmark JS SDK
+
+    `npm install bitmark-sdk`
```

Comment:
My personal preference is to clearly show you're issuing a UNIX command by including a command line, e.g.:
```
> npm install bitmark-sdk
```
Or:
```
# npm install bitmark-sdk
```
And this makes it easy to see outputs, as I'll talk about later.